packages/gasket-plugin-redux/lib/create.js

```diff
@@ -4,6 +4,8 @@ const { devDependencies } = require('../package.json');
 
 /** @type {import('@gasket/core').HookHandler<'create'>} */
 module.exports = async function create(gasket, context) {
+  if (Object.keys(context).includes('useRedux') && !context.useRedux) return;
```

Comment:
No sure that check is accurate. If `useRedux` is not on context, this check will fail and redux will be added.

More concise and accurate would be:
```suggestion
  if (context.useRedux !== true) return;
```

That is, if it's anything other than true, do not create with Redux